openshift.spec

```diff
@@ -18,11 +20,36 @@ Source0:        https://%{import_path}/archive/%{commit}/%{name}-%{version}.tar.
 BuildRequires:  systemd
 BuildRequires:  golang >= 1.2-7
 
-Requires:       docker-io >= 1.3.2
 
 %description
 %{summary}
 
+%package master
+Summary:        Openshift Master
+Requires:       openshift = %{version}-%{release}
+Requires:       openshift-sdn
+
```

Comment:
https://github.com/openshift/openshift-sdn is the project. There's a private fork where packaging work is in motion and those changes should be pushed upstream very soon. There should be dist-git for that as well, though I don't know if it has rhoso branches.

SDN is only necessary if you have more than two nodes. Perhaps we shouldn't enforce this here?